src/test/java/org/opendatakit/common/persistence/QueryResultTest.java

```diff
@@ -361,10 +363,20 @@ public void testCase3() throws ODKDatastoreException {
         int fetchSizes[] = {1, 2, 1021, 303, 101, 2831 };
         int idxFetch = 4;
         int len = 0;
-        QueryResult result = query.executeQuery(null, fetchSizes[idxFetch]);
+        // It seems that the following two asserts try to test if the results
+        // have less rows than the ones retrieved from the database by the query

```

Comment:
A period at the end of this sentence would be nice.
(I no longer worry about the less/fewer distinction.)